Dacaspex/Exhibit
# Feature/parse service

#### Deployment instructions
/

#### What have I done
fixes #149 
- Created the parse services that calls all file parses in order. 
- Created function to import a track, path and timing data from the front-end
    - Did not yet implement the more elaborate function to import more timing files, see #223 
    - Did not yet implement the functionality to import the 3D model, see #215 
- Hooked up import button with back-end.
- Changed implementation of parsers and dependent classes to be proper singletons without any parameters in the constructor call.

#### Testing
**Note:** The unit tests do not work because of a bug in Angular zone.js, see comment in test cases.
**Note:** This causes the code coverage to not be 100%, unfortunately

1. Go to the import screen `localhost:4200/import` and navigate to the import manually page (pressing the plus icon).
2. Select some files from Océ 
3. Observe the output in the console (issue #224 will implement an actual UI component for this error)
